core/src/ast_builder/select/root.rs

```diff
@@ -61,29 +63,63 @@ impl SelectNode {
         )
     }
 
-    pub fn left_join<'a>(self, table_name: &str) -> JoinNode<'a> {
+    pub fn left_join(self, table_name: &str) -> JoinNode<'a> {
         JoinNode::new(self, table_name.to_owned(), None, JoinOperatorType::Left)
     }
 
-    pub fn left_join_as<'a>(self, table_name: &str, alias: &str) -> JoinNode<'a> {
+    pub fn left_join_as(self, table_name: &str, alias: &str) -> JoinNode<'a> {
         JoinNode::new(
             self,
             table_name.to_owned(),
             Some(alias.to_owned()),
             JoinOperatorType::Left,
         )
     }
+
+    pub fn alias_as(self, table_alias: &'a str) -> TableAliasNode {
```

Comment:
This is too cool feature! Could we have an usage example in [test-suite/ ast_builder/select.rs](https://github.com/gluesql/gluesql/blob/main/test-suite/src/ast_builder/select.rs)?